src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useEffect, useState } from 'react';
+import { Filter } from '../../types/Filter';
+
+type Props = {
+  getQuery: (option: string) => void,
+  selectedFilter: (searchWord: string) => void,
```

Comment:
[Recommend reading this article](https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364)